src/test/regress/sql/explain_format.sql

```diff
@@ -9,13 +9,13 @@
 -- m/^Settings:.*/
 -- s/^Settings:[,\s]*/Settings: /
 -- m/^Settings:.*/
--- s/,?\s*optimizer\w*\s*=\s*'off'//g
+-- s/,?\s*optimizer\w*\s*=\s*'on'//g
```

Comment:
Maybe ignore `off` too?
```suggestion
-- s/,?\s*optimizer\w*\s*=\s*'(on|off)'//g
```

Reply:
I think we shouldn't try to catch 'off', as it is the default value now, and we expect that it shouldn't be printed in verbose mode.